buddyboss/buddyboss-platform
# Fixed - Album title removed when we upload errored photos

### All Submissions:


### Changes proposed in this Pull Request:

Fixes #1636 

### How to test the changes in this Pull Request:

1. Go to Album Create popup, type the title of the album
2. Upload some photos with more than the size limit provided by the server. usually 4 mb
3. Error will be shown about size close the popup of error.
4. See now that album title is not reset.


### Other information:
